app/components/land-use-map.js

```diff
@@ -1,12 +1,110 @@
 import Ember from 'ember'; // eslint-disable-line
 
 export default Ember.Component.extend({
-  geomStyle: {
-    fill: false,
-    color: '#000',
-    weight: 2,
-    dashArray: '8 5',
-    opacity: 0.6,
+  initOptions: {
+    style: 'mapbox://styles/mapbox/light-v9',
+    zoom: 9,
+    center: [-74, 40.7071],
+    scrollZoom: false,
   },
+
+  vectorSource: Ember.computed('mapState.landUseTemplate', function () {
```

Comment:
Good catch...  they used to be dependent on an async call to carto, now the URLs are static.